addons/mrp/models/stock_move.py

```diff
@@ -182,7 +182,7 @@ def _compute_should_consume_qty(self):
             if not mo:
                 move.should_consume_qty = 0
                 continue
-            move.should_consume_qty = mo.product_uom_id._compute_quantity((mo.qty_producing - mo.qty_produced) * move.unit_factor, mo.product_uom_id, rounding_method='HALF-UP')
+            move.should_consume_qty = move.product_uom._compute_quantity((mo.qty_producing - mo.qty_produced) * move.unit_factor, move.product_uom, rounding_method='HALF-UP')
```

Comment:
You can use `float_round` instead of `_compute_quantity` as we compute the quantity from one UOM to itself.
But in this case, add the `not move.product_uom` in the condition as this compute trigger for new line and if they haven't a an UOM, `float_round` will raise an error when checking the UOM's precision rounding. 
```python
            if not mo or not move.product_uom:
                move.should_consume_qty = 0
                continue
            move.should_consume_qty = float_round((mo.qty_producing - mo.qty_produced) * move.unit_factor, precision_rounding=move.product_uom.rounding)
```

Reply:
And so
```python
@api.depends('raw_material_production_id.qty_producing', 'product_uom', 'product_uom_qty')
```